sass/style.css

```diff
@@ -45,6 +57,33 @@ a {
   background: white;
 }
 
+.search-filter-container .bg-dark-blue {
+  padding: 2rem !important;
+}
+
+.search-filter-container h4 {
+  font-weight: 700;
+  font-size: 22px;
+}
+
+.search-filter-container p {
+  font-size: 16px;
+}
+
+.search-filter-container .input-group-text {
+  padding-right: .7rem !important;
+}
+
+.search-filter-container .form-control {
+  height: auto;
+  font-size: 14px;
+}
+
+/* even out the spacing between filters... */
+.filters-container > *:first-child {
```

Comment:
Would be better to add a class to that first col element and use that to set padding, instead of this selector + !important